src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.CallSiteContainerRewriter.cs

```diff
@@ -61,9 +61,13 @@ public override SyntaxNode VisitLocalDeclarationStatement(LocalDeclarationStatem
                 {
                     node = (LocalDeclarationStatementSyntax)base.VisitLocalDeclarationStatement(node);
 
+                    if (node.Declaration.IsDeconstructionDeclaration)
+                    {// PROTOTYPE(tuples)
```

Comment:
I wish to remove this marker before merging, but I'm not sure if additional scenarios need to be covered where variables need to be removed. 
I'll check with the @dotnet/roslyn-ide team.